### PR TITLE
feat(permissions): url changed telemetry

### PR DESCRIPTION
**Pull request by lisli1**
#### Description of changes

Added telemetry publishing for when the url of a tab changes

#### Pull request checklist

- [x] Addresses an existing issue: # 1645760
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). Check workflow guide at: `<rootDir>/docs/workflow.md`
- [n/a] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Comment by smoralesd** [comment]
There is a lint error on the build
```
ERROR: /Users/runner/runners/2.160.1/work/1/s/src/tests/unit/tests/background/actions/tab-action-creator.test.ts:15:1 - All imports on this line are unused.
```

File: src/common/extension-telemetry-events.ts [hunk 1]
```diff
@@ -60,6 +60,7 @@ export const VISUAL_HELPER_TOGGLED: string = 'visualHelperToggled';
 export const ALL_RULES_EXPANDED: string = 'allRulesExpanded';
 export const ALL_RULES_COLLAPSED: string = 'allRulesCollapsed';
 export const RESCAN_VISUALIZATION: string = 'rescanVisualization';
+export const EXISTING_TAB_UPDATED: string = 'existingTabUpdated';
```
Comment (smoralesd) [inline src/common/extension-telemetry-events.ts]: The new event we are adding on this PR is about the url being changed on an existing tab but then name of the event looks more generic than that. Would something like `'existingTabUrlUpdated'` be better?

File: src/tests/unit/tests/background/actions/tab-action-creator.test.ts [hunk 2]
```diff
@@ -1,16 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { PageVisibilityChangeTabPayload, SwitchToTargetTabPayload } from 'background/actions/action-payloads';
+import { ExistingTabUpdatedPayload, PageVisibilityChangeTabPayload, SwitchToTargetTabPayload } from 'background/actions/action-payloads';
 import { TabActionCreator } from 'background/actions/tab-action-creator';
 import { TabActions } from 'background/actions/tab-actions';
 import { TelemetryEventHandler } from 'background/telemetry/telemetry-event-handler';
 import { BrowserAdapter } from 'common/browser-adapters/browser-adapter';
-import { SWITCH_BACK_TO_TARGET, TelemetryEventSource, TriggeredBy } from 'common/extension-telemetry-events';
+import {
```
Comment (smoralesd) [inline src/tests/unit/tests/background/actions/tab-action-creator.test.ts]: there seems to be an extra, unneeded new line here.

Comment (lisli1) [reply src/tests/unit/tests/background/actions/tab-action-creator.test.ts]: This seems to be auto-formatted

**Pull request merged**